src/muxer/yamux/yamuxed_connection.cpp

```diff
@@ -40,7 +40,7 @@ namespace libp2p::connection {
         connection_{std::move(connection)},
         config_{config} {
     // client uses odd numbers, server - even
-    last_created_stream_id_ = connection_->isInitiator() ? 1 : 0;
+    last_created_stream_id_ = connection_->isInitiator() ? 1 : 2;
```

Comment:
why?

Reply:
StreamIds start from 1, not from 0, so the previous variant was incorrect